src/Resources/app/administration/src/module/sw-extension-store/service/extension-store-data.service.js

```diff
@@ -30,6 +30,21 @@ export default class ExtensionStoreDataService extends Shopware.Classes.ApiServi
         return extensions;
     }
 
+    /**
+     * @returns {Promise<Extension|null>}
+     */
+    async getExtensionByName(name, context) {
+        return this.getExtensionList({ term: name }, context).then((extensions) => {
+            for (let i = 0; i < extensions.length; i += 1) {
```

Comment:
What about `Array.find`? :sweat_smile:

 ```suggestion
           return extensions.find((extension) => { return extension.name === name; }) ?? null;
```

Reply:
No? :smile:

Reply:
yes!